RELEASE/scripts/autoscend/quests/level_any.ash

```diff
@@ -891,3 +891,53 @@ boolean LX_ForceNC()
 			return false;
 	}
 }
+
+boolean LX_dronesOut()
+{
+	if(!dronesOut())
+	{
+		return false;
+	}
+	location desiredDronesLocation = get_property("auto_dronesRouting").to_location();
+	if(desiredDronesLocation == $location[none]) return false;
+	boolean canExtingo = true;
+	if(auto_fireExtinguisherCharges() <= 30 || !canUse($skill[Fire Extinguisher: Polar Vortex], false))
+	{
+		canExtingo = false;
+	}
+
+	//where to go to. Not handling Smut Orc Keepsake, Blackberry Bush due to adventuring conditions required. If they happen to show up, they are handled in auto_combat
+	if(internalQuestStatus("questL04Bat") <= 1 && zone_isAvailable($location[The Batrat and Ratbat Burrow]))
+	{
+		return autoAdv($location[The Batrat and Ratbat Burrow]); //Sonar-in-a-Biscuit
+	}
+	if(item_amount($item[Stone Wool]) == 0 && have_effect($effect[Stone-Faced]) == 0 && canSummonMonster($monster[Baa\'baa\'bu\'ran]))

```

Comment:
there is no check that stone wools are needed here, won't this summon Baa\'baa\'bu\'ran every day after the hidden city was already unlocked?